meta-resin-common/recipes-bsp/u-boot/patches/env_resin.h

```diff
@@ -79,9 +83,17 @@
                        "run resin_import_env_file;" \
                "fi;\0" \
        \
+       "resin_check_kernel_extroot=" \
+               "if run resin_kernel_extdetect; then " \
+                 "env set resin_kernel_loader ext4load; " \
```

Comment:
What uses resin_kernel_loader ?

Reply:
Figured it out from the resin-rpi PR but I think we should have the information in commit log too.

Reply:
For history. https://github.com/resin-os/resin-raspberrypi/pull/225#discussion_r214991744